daprdocs/content/en/getting-started/install-dapr-selfhost.md

```diff
@@ -10,6 +10,8 @@ aliases:
 
 Now that you've [installed the Dapr CLI]({{<ref install-dapr-cli.md>}}), use the CLI to initialize Dapr on your local machine.
 
+To initialize dapr in your local (or remote) kubernetes cluster. [How to initialize dapr with redis and zipkin]({{<ref "kubernetes-deploy.md#install-dapr-from-the-official-dapr-helm-chart-with-development-flag">}})
```

Comment:
```suggestion
To initialize Dapr in your local (or remote) kubernetes cluster for development including the Redis and Zipkin containers listed above see [how to initialize Dapr for development on Kubernetes]({{<ref "kubernetes-deploy.md#install-dapr-from-the-official-dapr-helm-chart-with-development-flag">}})
```

Reply:
I suggest that this paragraph above ^^ be move to after this line in the doc

"4. Running a Dapr placement service container instance for local actor support."

That way you have some understand of what will happen first with Zipkin and Redis.

Reply:
@msfussell I'm thinking that maybe keeping it where it is but putting it into an alert with the kubernetes title may be a better fit as this diverges from the rest of the guide and should provide a helpful hint towards the kubernetes init process

Reply:
@mikeee  -given the text where is says deploys zipkin and Redis, it does fit better in an alert further down the page. I move it there, in an edit, PTAL and tell me. IMO this is good to merge now